src/cr/cube/matrix/assembler.py

```diff
@@ -151,22 +151,13 @@ def columns_dimension_numeric_values(self):
     def columns_margin(self):
         """1D/2D np.float64 ndarray of weighted-N for each column of this slice."""
         # --- an MR_X slice produces a 2D columns-margin (each cell has its own N) ---
+        # --- This is really just another way to call the columns_weighted_bases ---
+        # --- TODO: Should this error for 2D margins? ---
```

Comment:
We are considering making `columns_margin` always be 1D, and error on arrays. Then having a `columns_margin_range` that would be used for array variables. This would require more changes to exporter though

Reply:
Can you write this more meaningful explanation as the TODO ? If we read at this comment in 2 months we propably have forgotten the rationale :D

Reply:
Can you write this more meaningful explanation as the TODO ? If we read at this comment in 2 months we propably have forgotten the rationale :D